### ISSUE TITLE
Icon Request: microsoft teams

### ISSUE THREAD

**Issue by maxklock** [FortAwesome/Font-Awesome#10704]
![image](https://cloud.githubusercontent.com/assets/7167301/23339717/53e2a288-fc28-11e6-819e-f724fcdc2642.png)
(from dev.botframework.com)

I want to use it, to add a "add to" link for my Chat bot, developed with the Microsoft Botframework. But It could also be used to tell others, that you're using Microsoft Teams

**Comment by JustinObm** [comment]
With everyone working from home now, MS Teams is being used more and more, I made my own SVG so that staff can click a link in our Staff Directory to open a chat with them, would be nice to have a font though. (https://blog.thoughtstuff.co.uk/2019/02/how-to-use-a-hyperlink-to-jump-directly-to-a-teams-private-chat-with-a-user/)

Skype for Business will reach end of life in July 2021, Teams is the new replacement.

**Comment by fulviomedeot** [comment]
+1

**Comment by alex-woodall-premierfoods** [comment]
+1

**Comment by var-greyShader** [comment]
Yes, it would be appreciable if you guys add Microsoft teams to the collection

**Comment by jamespavett** [comment]
Definitely needing after the increase uptake of teams recently!

**Comment by samdkershaw** [comment]
+1

**Comment by jojofromkokomo** [comment]
+1

**Comment by asbcoder** [comment]
+ 1

**Comment by andyr79** [comment]
+1

**Comment by rudacs** [comment]
+1

**Comment by alexFortuna3** [comment]
+1

**Comment by warrenberberd** [comment]
+1

**Comment by tanluongann** [comment]
+1

**Comment by quesurifn** [comment]
+1

**Comment by FerringNL** [comment]
+1

**Comment by GregoireLgt** [comment]
+1

**Comment by jonny64bit** [comment]
+1

**Comment by debasisdev** [comment]
+1200

**Comment by antenucci96** [comment]
+1

**Comment by rubendgpedrosa** [comment]
+1

**Comment by jhumphries83** [comment]
+1

**Comment by radloffl** [comment]
Bump

**Comment by S33G** [comment]
bump!

**Comment by faisalnjs** [comment]
bump

**Comment by NeverFearAThing** [comment]
+1

**Comment by alonsomoya** [comment]
+1

**Comment by antonylineesh1** [comment]
+1

**Comment by MudulOzan** [comment]
bump

**Comment by fireballhh** [comment]
Would definetly be great to have for my sites +1

Why got the new icon label removed? There is no Teams icon right? So its new or not? :D

**Comment by ddvdv** [comment]
+1

**Comment by haenno** [comment]
+1

**Comment by KaiKorla** [comment]
+1

**Comment by gnuheidix** [comment]
+1

**Comment by eirikedh** [comment]
+1

**Comment by megaroeny** [comment]
+1 please. And the original icon/logo in the issue is outdated now.

**Comment by podtris** [comment]
+1 please

**Comment by mlechler** [comment]
+1 please

**Comment by imuller** [comment]
Yeah please!

**Comment by FelipeIGF** [comment]
Skype will be replaced by Teams soon

**Comment by tutopasini** [comment]
+1

**Comment by joaoschumacher** [comment]
+1

**Comment by GabrielPdaC** [comment]
+1

**Comment by Ronnau** [comment]
+1

**Comment by BernardoLcs** [comment]
+1

**Comment by gustavosimon** [comment]
+1

**Comment by Edsonjr05** [comment]
+1

**Comment by cristianms** [comment]
+1

**Comment by arthurfmalaq** [comment]
+1

**Comment by MarhlonKorb** [comment]
+1

**Comment by carlosfavero** [comment]
+1

**Comment by Cippher** [comment]
+1

**Comment by cassioquint** [comment]
+1

**Comment by fernandacarolinesantos** [comment]
+1

**Comment by Diosef** [comment]
+1

**Comment by wagnerjk** [comment]
+1

**Comment by Lucas-Camargo001** [comment]
+1

**Comment by dazzimachado** [comment]
+1

**Comment by Matheus-Camargo-Rech** [comment]
+1

**Comment by rafahx** [comment]
+1

**Comment by chenchunk** [comment]
+1

**Comment by paulkakell** [comment]
+1

**Comment by Pinkeltje** [comment]
+1

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.

**Comment by megaroeny** [comment]
Done! And no worries, glad you got a better system going!

https://github.com/FortAwesome/Font-Awesome/discussions/21035
